docs/release-notes/highlights-2.14.0.asciidoc

```diff
@@ -1,6 +1,24 @@
 [[release-highlights-2.14.0]]
 == 2.14.0 release highlights
 
+[[known-issue-2.14.0]]
+[float]
+=== Known issue
+Users who have defined a <<{p}-readiness,custom readiness probe>> for Elasticsearch 8.2.0 or later will have to either remove the custom readiness probe after upgrading to 2.14 or have adjust the readiness probe script as follows:
```

Comment:
shouldn't the customer remove the readiness probe before upgrading. to 2.14 ?

Reply:
I thought about it as well. I assume customers overriding the probe have clusters under heavy load that would become unready if the probe was removed before the upgrade. Whereas after the upgrade a limited number of nodes would be affected. So it's probably a decision dependent on the particular situation. I can try to reword it a bit to make it clearer.

Reply:
makes sense